tests/unit/models/product-model-test.js

```diff
@@ -95,3 +95,9 @@ test('it attaches a reference to the config on variants', function (assert) {
     assert.equal(variant.config, config);
   });
 });
+
+test('it returns Shopify admin\'s no image URI', function (assert) {
+  assert.expect(1);
+
+  assert.equal(NO_IMAGE_URI, 'https://widgets.shopifyapps.com/assets/no-image.svg');
+});
```

Comment:
I would just test that this a string of non-zero length. I don't think we should be validating what the actual URI is.